pkg/download/goget/goget_test.go

```diff
@@ -35,3 +43,19 @@ func TestList(t *testing.T) {
 		})
 	}
 }
+
+func TestInvalidGoBinError(t *testing.T) {
```

Comment:
we don't need this test, because we are unit testing `TestGoGetFetcherError` which takes care of this scenario.

Reply:
This test would ensure we're propagating the error from the call.

Reply:
@devdinu it's not necessary to test that. If we're not propagating en error, that means we're not checking it. If we're not checking, that means we're doing something seriously wrong and it's ok to panic.